jsonpatcher/jsonpatcher.go

```diff
@@ -252,4 +212,8 @@ func (je patchEvent) Model() string {
 	return je.ModelName
 }
 
+func (je patchEvent) Type() core.ActionType {
```

Comment:
This has been added to the core Event interface.

Reply:
+1

Reply:
Ok, so what's a viable alternative here then? Thoughts?